sktime/benchmarking/benchmarks.py

```diff
@@ -10,6 +10,39 @@
 from sktime.utils.validation._dependencies import _check_soft_dependencies
 
 
+def coer_estimator_and_id(estimators, estimator_id=None):
+    """Coerce estimators to a dict with estimator_id as key and estimator as value.
+
+    Parameters
+    ----------
+    estimators : dict, list or BaseEstimator object
+        Estimator to coerce to a dict.
+    estimator_id : str, optional (default=None)
+        Identifier for estimator. If none given then uses estimator's class name.
+
+    Returns
+    -------
+    estimators : dict
+        Dict with estimator_id as key and estimator as value.
+    """
+    VERSION_ID = "-v1"
+    if isinstance(estimators, dict):
```

Comment:
Is it worth checking that all values are `BaseEstimator` objects, as per docstring?